apps/minting-app/components/color-picker/index.tsx

```diff
@@ -185,3 +194,27 @@ function fromLightness(value: number): number {
 
   return progress * 180;
 }
+
+type ThrottledFunction<T extends (...args: any) => void> = (
```

Comment:
Why not lodash/throttle?

Reply:
simple enough function to just copy rather than bundling a whole lot of bullshit from lodash deps... also it's improved over lodash's implementation by using `requestIdleCallback`